config/config.go

```diff
@@ -122,8 +124,8 @@ type VochainCfg struct {
 	SkipPreviousOffchainData bool
 	// Enable Prometheus metrics from tendermint
 	TendermintMetrics bool
-	// Target block time in seconds (only for miners)
-	MinerTargetBlockTimeSeconds int
+	// Target block time (only for miners)
+	MinerTargetBlockTime time.Duration
```

Comment:
This change might be breaking. We should check node configurations out there that needs to be changed.

Reply:
i've removed this change, and some others that were not essential to the fix.